releasenotes/notes/nova-13.0-0c2497128e677cb2.yaml

```diff
@@ -0,0 +1,20 @@
+---
+prelude: >
+    This version introduces nova's new database,
+    nova api db.
+security:
```

Comment:
I'm not sure about what you mean: You think the nova api db introduction should be in security?
What about the speed? We now are more secure. Unless the deployer explicitly decides to do so (then the information in other could be useful). I can maybe merge the note about the live migration in the security - and mentioning it would be insecure.